lib/src/analyzers/lint_analyzer/rules/rules_list/prefer_const_border_radius/visitor.dart

```diff
@@ -0,0 +1,25 @@
+part of 'prefer_const_border_radius.dart';
+
+class _Visitor extends RecursiveAstVisitor<dynamic> {
+  final _declarations = <AstNode>[];
+
+  Iterable<AstNode> get declarations => _declarations;
+
+  @override
+  void visitTypeName(TypeName node) {
```

Comment:
Have you checked other options before choosing TypeName? It looks like we can use something like a constructor call which will trigger less in average AST.

Reply:
Dima suggested a solution through `visitInstanceCreationExpression`